.circleci/config.yml

```diff
@@ -60,6 +60,10 @@ jobs:
       docker_layer_caching: true
     steps:
       - checkout
+      - run:
+          name: Checkout submodules
+          command: |
+            git submodule update --init --recursive
```

Comment:
There needs to be a corresponding command somewhere in ./gui.sh so that ./gui.sh --build runs this. I spent some time trying to figure out why running it on this PR didn't build ProjectVentilatorGUI anymore - turned out cause it didn't have nanopainter, and what's worse, also didn't report any error from gui.sh --build.

Might actually be easier to just check in a particular full version of the library.